sources/packages/web/src/views/institution/locations/active-applications/LocationActiveApplicationSummary.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <v-container>
+    <p class="text-muted font-weight-bold h3">{{ locationName }}</p>
+    <p class="font-weight-bold h2">Active Applications</p>
+    <v-sheet elevation="1" class="mx-auto mt-2">
+      <v-container>
+        <DataTable
+          :autoLayout="true"
+          :value="applications"
+          class="p-m-4"
+          :paginator="true"
+          :rows="10"
+        >
+          <Column field="firstName" header="Name">
+            <template #body="slotProps">
+              <span
+                >{{ slotProps.data.firstName }}
```

Comment:
There is a method in the API that can handle the proper concatenation of first name and last name.
Sometimes the first name could be null, for instance, for mononymous names.
I would recommend returning this properly formatted from the API for this particular case. Case this was based on an existing code, it could also be refactored (if you would be willing to d it 😉 ).
The name of the method on the API is _getUserFullName_.